Userland/Libraries/LibWeb/Page/EventHandler.cpp

```diff
@@ -1023,6 +1023,22 @@ EventResult EventHandler::handle_keydown(UIEvents::KeyCode key, u32 modifiers, u
             FIRE(input_event(UIEvents::EventNames::input, UIEvents::InputTypes::insertText, m_navigable, code_point));
             return EventResult::Handled;
         }
+    } else if (auto selection = document->get_selection(); selection && !selection->is_collapsed()) {
+        if (modifiers & UIEvents::Mod_Shift) {
```

Comment:
Huh how do we end up here? Seems weird that we'd have no event target

Reply:
we end up here when selection is located outside of input element. maybe I should rename `target` to `input_event_target` to make it more explicit.

Reply:
Ah I see, makes sense - I missed/forgot that we get `target` from `active_input_events_target`